driver-core/src/test/functional/com/mongodb/internal/connection/ServerMonitorSpecification.groovy

```diff
@@ -103,96 +103,96 @@ class ServerMonitorSpecification extends OperationFunctionalSpecification {
         ServerDescription otherDescription
 
         expect:
-        !shouldLogStageChange(description, builder.build())
+        !shouldPublishChangeEvent(description, builder.build())
```

Comment:
Seems like this test no longer belongs in this file.  Consider a new test file for the helper class

Reply:
I have moved the test to ServerDescriptionChangeEventHelperSpecification.groovy.